dev-docs/multipart-subscriptions-protocol.md

```diff
@@ -43,13 +43,11 @@ Content-Type: application/json
 --graphql--
 ```
 
-## Messages
+## GraphQL Responses
 
-The protocol differentiates between transport-level concerns and the GraphQL response payloads themselves. One reason for this is that [the response format is part of the GraphQL spec](https://spec.graphql.org/draft/#sec-Response-Format), and additional fields might be confusing or could even break client typing.
+When a GraphQL response is received, it is returned in the `payload` field using the [format from the GraphQL spec](https://spec.graphql.org/draft/#sec-Response-Format).
 
-Except for [heartbeats](#heartbeats), every message will therefore include a `payload`, but the payload may be `null`.
-
-Some errors are part of an execution result. Results may include partial data, and these errors are not fatal (meaning the subscription stream should be kept open). They are therefore delivered within the payload:
+A GraphQL response may contains [errors](https://spec.graphql.org/October2021/#sec-Errors). These errors are not fatal (meaning the subscription stream should be kept open) and are delivered within the payload:
```

Comment:
Only saw this on the fifth read through 🙃

```suggestion
A GraphQL response may contain [errors](https://spec.graphql.org/October2021/#sec-Errors). These errors are not fatal (meaning the subscription stream should be kept open) and are delivered within the payload:
```

Reply:
Thanks